src/main/kotlin/biokotlin/cli/GetVCFVariantsExample.kt

```diff
@@ -0,0 +1,84 @@
+package biokotlin.cli
+
+import biokotlin.util.GetVCFVariants
+import biokotlin.util.SimpleVariant
+import biokotlin.util.getAllVCFFiles
+import biokotlin.util.validateVCFs
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.Channel
+import org.apache.logging.log4j.LogManager
+
+/**
+ * This is an example use of GetVCFVariants.
+ * See MergeGVCFs.kt for a similar example.
+ */
+class GetVCFVariantsExample : CliktCommand(help = "Example use of GetVCFVariants.") {
+
+    private val myLogger = LogManager.getLogger(GetVCFVariantsExample::class.java)
+
+    val inputDir by option(help = "Full path to input VCF file directory")
+        .required()
+
+    // I put Any?, but you should replace it with the actual type of the result
+    private val resultChannel = Channel<Deferred<Any?>>(100)
```

Comment:
Related to comment above - who would use this based on the comment at line 25?  Is this an example for developers?  Should it stay in Biokotlin and not here?  I might be missing something.

Reply:
@lynnjo No users do not call this. It is an example for people using GetVCFVariants utility

Reply:
@lynnjo This class is for developers

Reply:
Should it be in BioKotlinTools?  I guess since it isn't in BiokotlinTools.kt users won't know about it so it shouldn't be confusing.  Do we expect it to become a command for users in the future?

Reply:
I agree with Lynn, there are better places to put something like this and they should be in biokotlin specifically.  I am not sure you need the Clikt interface to wrap how to parse the GVCFs